### PR TITLE
Prevent copying proposals.

### LINKED ISSUE #2282: Attempting to copy dossier with proposal looks for non-existent SQL model
Attempting to copy a dossier with a proposal will look for an SQL proposal model by OGUID that doesn't exist.

**Steps to reproduce:**
- Copy the existing example content dossier http://localhost:8080/fd/ordnungssystem/ressourcen-und-support/personal/personalrekrutierung/dossier-4
- Attempt to paste in a different repository folder

This results in:

```
2016-11-11 09:25:19 ERROR Zope.SiteErrorLog 1478852719.420.885242387786 http://localhost:8080/fd/ordnungssystem/ressourcen-und-support/personal/personaladministration/paste_clipboard
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module grokcore.view.components, line 151, in __call__
  Module zope.publisher.publish, line 107, in mapply
   - __traceback_info__: <bound method PasteClipboardView.render of <opengever.base.browser.paste.PasteClipboardView object at 0x118e2eb90>>
  Module zope.publisher.publish, line 113, in debug_call
  Module opengever.base.browser.paste, line 44, in render
  Module opengever.base.browser.paste, line 71, in copy_objects
  Module opengever.base.browser.paste, line 74, in rename_object
  Module opengever.base.browser.paste, line 77, in _recursive_rename
  Module plone.api.content, line 2, in rename
  Module plone.api.validation, line 70, in wrapped
  Module plone.api.content, line 222, in rename
  Module plone.folder.ordered, line 187, in manage_renameObject
  Module OFS.CopySupport, line 392, in manage_renameObject
  Module zope.event, line 31, in notify
  Module zope.component.event, line 24, in dispatch
  Module zope.component._api, line 136, in subscribers
  Module zope.component.registry, line 321, in subscribers
  Module zope.interface.adapter, line 585, in subscribers
  Module zope.component.event, line 32, in objectEventNotify
  Module zope.component._api, line 136, in subscribers
  Module zope.component.registry, line 321, in subscribers
  Module zope.interface.adapter, line 585, in subscribers
  Module OFS.subscribers, line 113, in dispatchObjectMovedEvent
  Module zope.container.contained, line 153, in dispatchToSublocations
  Module zope.component._api, line 136, in subscribers
  Module zope.component.registry, line 321, in subscribers
  Module zope.interface.adapter, line 585, in subscribers
  Module opengever.meeting.handlers, line 69, in sync_moved_proposal
  Module opengever.meeting.proposal, line 484, in sync_model
AttributeError: 'NoneType' object has no attribute 'language'
```

This seems to happen because `load_model()` will look for a proposal in SQL by OGUID, with an OGUID that doesn't exist in SQL yet. 

Also see: https://extranet.4teamwork.ch/support/0339-support-kg-koeniz/0339-support-onegov-gever-evangelisch-reformierte-kirchgemeinde-koeniz/46

### PR DESCRIPTION
**Pull request by deiferni**
Proposals should not be copied when copying a dossier.

There are many states where this does not make sense at the moment or the desired state after copying is unclear (e.g. already submitted or decided proposals.) Thus we prevent them from being copied for now.

Deleting the proposal right after it had been created had some consequences for the other event handlers, that's why i had to change from acquiring request/catalog to getting it through a global accessor.

Fixes #2282.

### CODE REVIEW

File: opengever/meeting/handlers.py [hunk 1]
```diff
@@ -60,8 +66,26 @@ def _sync_excerpt(document):
...
+    Turns out the easiest way to accomplish this is to delete the proposal
+    after it has been copied.
+    """
+    with elevated_privileges():
+        api.content.delete(obj)
+
+
 @grok.subscribe(IProposal, IObjectMovedEvent)
 def sync_moved_proposal(obj, event):
+    if IObjectAddedEvent.providedBy(event):
```
Comment (phgross) [inline opengever/meeting/handlers.py]: Why is that necessary, please add a comment

**Comment by deiferni** [comment]
@phgross updated
